docs/reference-guides/best-practices/rancher-server/tips-for-scaling-rancher.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Tips for Scaling Rancher
+---
+
+This guide aims at introducing challenges and approaches that should be considered to scale Rancher setups. As setups grow performance will reduce. There are steps we can take to minimize the load that is put on Rancher and optimize Rancher's ability to handle increased loads.
+
+## General Tips on Optimizing Rancher's Performance
+* It is advisable to keep Rancher up to date with patch releases. Performance improvements and bug fixes are made throughout the life of a minor release. You can review the release notes to help inform your own decisions on whether an upgrade is necessary but we recommend keeping yourself up to date in most cases.
+
+* The geographically closer Rancher's infrastructure and downstream cluster's infrastructure are the better. If a user or organization requires clusters/nodes all over the world or spread across many regions, it is best to use multiple Rancher installations.
+
+## Minimizing Load on the local cluster
+The largest bottleneck when scaling Rancher is kubernetes resource growth in the local cluster. The local cluster contains information for all downstream clusters. Many operations that apply to downstream clusters will create new objects in the local cluster and require computation from handlers running in the local cluster.
+
+### Managing Your Object Counts
+ETCD eventually encounters limitations to the number of a single kubernetes resource type it can store. These exact numbers are not well documented. From internal observations we usually see performance issues once an object count exceeds 60k. Usually the kubernetes resource type that reaches these numbers is Rolebinding.
+
+Rolebindings are created in the local cluster as a side effect of many operations. This includes creating clusters, adding users, adding projects, and adding roles to users. When adding users to cluster or projects, rolebindings will be created.
+
+Considerations when attempting reduce rolebindings in the local cluster:
+* Only add users to clusters and projects when necessary
+* Remove clusters and projects when they are no longer needed
+* Only use custom roles if necessary
+* Use as few rules as possible in custom roles
+* Consider whether adding a role to a user is redundant
+* Consider that using less, but more powerful, clusters may be more efficient
```

Comment:
```suggestion
* Consider that using fewer powerful downstream clusters will be more efficient
```

Reply:
I think that the revised version my be interpreted differently then intended. That could mean, use less powerful cluster than you are currently using instead of use less clusters but more powerful ones.

Reply:
Good point